src/main/java/quickndirty/minisurveymonkey/SurveyRepository.java

```diff
@@ -6,6 +6,6 @@
 public interface SurveyRepository extends JpaRepository<Survey, Integer>{
 
     //find by Name
-    public Survey findByName();
+    public Survey findByName(String name);
```

Comment:
This was causing errors before that would stop the app from starting. It runs now, but I think this might be missing an annotation though. Does anyone know?

Reply:
@RepositoryRestResource(collectionResourceRel = "survey", path = "survey")
possibly this. but if its working we can worry about it later

Reply:
Ah yeah I found this:
```
@RepositoryRestResource(collectionResourceRel = "student", path = "student")
public interface StudentRepository extends PagingAndSortingRepository<Student, Long> {
    Student findByName(@Param("name") String name);
}
```
I think that's what we need. I'll update the PR.

Reply:
Okay, I updated all 3 of the repository interfaces we have now to use the `RepositoryRestResource` annotation. @nikolaerak, @Aleksandar1231 does this make sense?